release-please-config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "release-type": "node",
+  "include-component-in-tag": false,
+  "packages": {
+    ".": {
+      "extra-files": [
```

Comment:
What about docker build and version? This includes the images version in the `values` file.
We could say it dosen't belong here but in a `docker-release` workflow, but then `release-please` should be added to there as well.

Reply:
I commented the tag so appVersion will be taked